app/views/collections/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="container" id="collection" data-controller="collection-updates" dat-collection-updates-collection-id="<%= @collection.id %>">
+  <%= render 'nav' %>
+  <section class="details">
+    <header class="title">
+      <span class="header-text"><%= @collection.name %></span>
+      <span class="deposit-button">
+        <% if allowed_to?(:deposit?, @collection) %>
```

Comment:
This is not a rule we have.  It should use `allowed_to?(:create?, Work.new(collection: @collection))`